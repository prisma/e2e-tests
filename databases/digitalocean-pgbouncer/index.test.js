const {
  clientWithoutQueryStringParamCall,
  clientWithQueryStringParamCall,
  client,
  clientWithQueryStringParam,
} = require('.')

describe('should test Prisma client and PgBouncer', () => {
  afterAll(async () => {
    await client.disconnect()
    await clientWithQueryStringParam.disconnect()

  })

  it('should work with default do pgbouncer without the pgbouncer query string param', async () => {
    const data = await clientWithoutQueryStringParamCall()
    expect(data).toMatchSnapshot()
  })

  it('should work with default do pgbouncer with the pgbouncer query string param', async () => {
    const data = await clientWithQueryStringParamCall()
    expect(data).toMatchSnapshot()
  })
})
