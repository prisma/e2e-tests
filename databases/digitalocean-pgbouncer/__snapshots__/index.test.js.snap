// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Prisma client and pgBouncer should work with default do pgBouncer with the forcedTransactions flag 1`] = `
Array [
  Object {
    "email": "alice-d97227bb-97b6-4a6d-81bf-04322b686165@prisma.io",
    "id": "d97227bb-97b6-4a6d-81bf-04322b686165",
    "name": "Alice",
  },
]
`;

exports[`should test Prisma client and pgBouncer should work with default do pgBouncer without the forcedTransactions flag 1`] = `
Array [
  Object {
    "email": "alice-d97227bb-97b6-4a6d-81bf-04322b686165@prisma.io",
    "id": "d97227bb-97b6-4a6d-81bf-04322b686165",
    "name": "Alice",
  },
]
`;
